### DISCUSSION TITLE
set custom default terminal/command line

### DISCUSSION THREAD

**Discussion by verajosemanuel** [adirh3/Fluent-Search#1097] [Q&A]
i use a lot the terminal and the feature "open file/folder" in command line would be amazing if i could set the default cmd instead of PowerShell (git bash)

is that possible?

**Comment by adirh3** [comment]
Yes this is possible by setting a favorite command line in Settings -> Command -> Command line interfaces.

**Reply by verajosemanuel** [reply]
Ok But I get an error regarding the folder path inside brackets

**Reply by adirh3** [reply]
Which brackets? Can you share screenshot of your settings and the error

**Reply by verajosemanuel** [reply]
open folder "data" then select menu option "open directory in command line" (git bash is the default profile in Windows Terminal)

[error 2147942402 (0x80070002) when launching `cd 'C:\Users\joe\data'']

configuration is:
name: wt
cmd: gitbash
executable path: C:\Users\joe\AppData\Local\Microsoft\WindowsApp\wt.exe
